recipe/meta.yaml

```diff
@@ -205,6 +216,12 @@ outputs:
         - test -f ${PREFIX}/lib/libglib-2.0${SHLIB_EXT}             # [unix]
         - glib-compile-resources --help
         - gobject-query --help
+        # I'm not sure why the PREFIX ends up becoming the /usr/bin/env python
```

Comment:
Shebang's can't be longer than 80 lines, so conda fixes them for you.

Reply:
ah i see. Thanks!

In this case, is the test correct, or should it be made stricter? (i.e. only testing for /usr/bin/env python)?

Reply:
Hmm ... did you check whether these tests fail if you don't apply the shebang fix? I am pretty sure that if the shebang line in the source file had `$BUILD_PREFIX` in it, the test would indeed fail, because Conda's prefix-rewriting logic wouldn't be activated at all.

If that's the case, then I think the test is fine as-is. I believe that in all of our build environments, the package test is run inside a directory that's got a sufficiently long absolute path that the /usr/bin/env replacement will always be active. But if you build this recipe outside of the conda-forge framework, the test prefix could conceivably be something like `/c/conda-bld/glib-123556890/_test_env` where the rewriting would *not* kick in.

Isuru's diagnosis is 100% correct, I believe, so it might be good to update the comment and add a link to this pull request.

Reply:
ok lets test the tests. I think they worked in the past. but i forget.

Reply:
Oooohhhhh that's tricky — it looks like the YAML parse is yielding surprising results for those lines. If they were passed through to the shell verbatim, I think they would work as you had them before. It might be worth encasing the whole lines in double quotes to make the YAML-level subtlety more apparent.

Reply:
I couldn’t get the matching of the # to work. But matching it with anything helped test for the prefix. 

Now let’s see if the “fix” works